lib/beaker/hypervisor/vagrant.rb

```diff
@@ -53,7 +53,11 @@ def make_vfile hosts, options = {}
 
         unless host['mount_folders'].nil?
           host['mount_folders'].each do |name, folder|
-            v_file << "    v.vm.synced_folder '#{folder[:from]}', '#{folder[:to]}', create: true\n"
+            unless folder[:from].nil? or folder[:to].nil?
```

Comment:
@cardil this line is a bit hard to understand & read quickly, regardless of its correctness. If you would change this to
```ruby
if !folder[:from].nil? && !folder[:to].nil?
```
this would be much easier to deal with.

Reply:
Hmm, I propose to write this like that:

```ruby
mount_folder = MountFolder.new(name, folder)
if mount_folder.required_keys_present?
  v_file << mount_folder.vfile_instructions
else
  @logger.warn "[...]"
end
```

What do you think?

Reply:
@cardil sure, that object creation sounds fine to me.